docs/en/notes/compatibility.md

```diff
@@ -186,7 +186,7 @@ We adopt new pre-processing and conversion steps of ScanNet dataset. In previous
 
 - There is an `PointSample` pipeline in the data pipelines for ScanNet detection task which down-samples points. So removing down-sampling in data generation will not affect the code.
 
-We have trained a [VoteNet](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/votenet/votenet_8x8_scannet-3d-18class.py) model on the newly processed ScanNet dataset and get similar benchmark results. In order to prepare ScanNet data for both detection and segmentation tasks, please re-run the new pre-processing scripts following the ScanNet [README.md](https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/).
+We have trained a [VoteNet](https://github.com/open-mmlab/mmdetection3d/blob/main/configs/votenet/votenet_8x8_scannet-3d-18class.py) model on the newly processed ScanNet dataset and get similar benchmark results. In order to prepare ScanNet data for both detection and segmentation tasks, please re-run the new pre-processing scripts following the ScanNet [README.md](https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/).
```

Comment:
votenet_8x8_scannet-3d-18class -> votenet_8xb8_scannet-3d.py

https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/ -> https://github.com/open-mmlab/mmdetection3d/blob/main/data/scannet/README.md/

Reply:
This is the description of version 0.14. We should not have modified `master` to `main`. I revert it.